php-practice/section7.php

```diff
@@ -36,6 +39,16 @@
  * ($string1 and $string2) and returns the combined length of both of them.
  */
 
+	function lengthOfStrings($string1, $string2) {
+		$combinedLength = ($string1.$string2);
+
+		echo strlen($combinedLength).PHP_EOL;
+};
+
+lengthOfStrings('Paul', 'Russell');
```

Comment:
Good job. Missed one detail -- this function is supposed to **return** the combined length, not print it out. (Although, it does print out the correct length. So good job!)

Some really concise ways of doing this would just be:

```php
function lengthOfStrings($string1, string2) {
  return strlen($string1 . $string2);
}
```

...or...

```php
function lengthOfStrings($string1, string2) {
  return strlen($string1) + strlen($string2);
}
```